add package: StickWithMarkdownSnippets

UniFreak opened the pull request:
<!--
Your pull request will be reviewed automatically and by a human.

The manual review may take several days or weeks, depending on the reviewer's availability and workload.
If you haven't received a comment on your pull request and it wasn't merged either,
it just hasn't been reviewed yet.

---

Please ensure the automated reviews pass. 
Follow the instructions provided, if necessary.
You can speed up the process
by [running some tests locally](https://packagecontrol.io/docs/submitting_a_package#Step_7).

You can trigger @packagecontrol-bot to re-evaluate your pull request
by pushing a commit or closing and reopening your pull request.
Do **NOT** open a new pull request!

In general, make sure you:

 1. Used `"tags": true` and not `"branch": "master"` 
    (versioning docs: <https://packagecontrol.io/docs/submitting_a_package#Step_4>)
 2. Added a README to your repository so that users (and reviewers) 
    can understand what your package provides.
 
You should proceed with a short description of what the package does
and, in case one or multiple similar package already exists, 
why you believe it is different and needed
below this line. -->

FichteFoll commented:
You don't need `.no-sublime-package`. Please remove it.

You can't access `sublime`'s API at module load time ([with exceptions](https://www.sublimetext.com/docs/3/api_reference.html#plugin_lifecycle))

Instead of `sublime.load_settings('Preferences.sublime-settings')`, use `view.settings()`, which ST will resolve automatically and accounts for view-specific settings.

UniFreak commented:
@FichteFoll Thanks for the tip! I have changed my plugin according to your suggestions, and made the following edits:
- removed `.no-sublime-package` file
- load settings in `plugin_loaded()` function
- replace `sublime.loading_settings()` call with `view.settings()`

I have made a new commit, please review it :)
